tests/integration-tests/reports/reportsApi-test.js

```diff
@@ -624,6 +624,69 @@ describe('Integration tests for the reports api', function () {
+
+                const statDateFirst = new Date(testStartTime).setSeconds(testStartTime.getSeconds() + 20);
+                let intermediateStatsResponse = await reportsRequestCreator.postStats(testId, reportId, statsGenerator.generateStats('intermediate', runnerId, statDateFirst, 600));
+                should(intermediateStatsResponse.statusCode).be.eql(204);
+                
+
+                const statDateSecond = new Date(testStartTime).setSeconds(testStartTime.getSeconds() + 40);
+                intermediateStatsResponse = await reportsRequestCreator.postStats(testId, reportId, statsGenerator.generateStats('intermediate', runnerId, statDateSecond, 200));
+                should(intermediateStatsResponse.statusCode).be.eql(204);
+                
+
+                const statDateThird = new Date(testStartTime).setSeconds(testStartTime.getSeconds() + 60);
+                intermediateStatsResponse = await reportsRequestCreator.postStats(testId, reportId, statsGenerator.generateStats('intermediate', runnerId, statDateThird, 200));
+                should(intermediateStatsResponse.statusCode).be.eql(204);
+                
+
+                const statDateFourth = new Date(testStartTime).setSeconds(testStartTime.getSeconds() + 80);
+                intermediateStatsResponse = await reportsRequestCreator.postStats(testId, reportId, statsGenerator.generateStats('intermediate', runnerId, statDateFourth, 200));
+                should(intermediateStatsResponse.statusCode).be.eql(204);
+                
+
+                const statDateFifth = new Date(testStartTime).setSeconds(testStartTime.getSeconds() + 100);
+                intermediateStatsResponse = await reportsRequestCreator.postStats(testId, reportId, statsGenerator.generateStats('intermediate', runnerId, statDateFifth, 200));
+                should(intermediateStatsResponse.statusCode).be.eql(204);
+                
+
+                const statDateSixth = new Date(testStartTime).setSeconds(testStartTime.getSeconds() + 120);
+                const doneStatsResponse = await reportsRequestCreator.postStats(testId, reportId, statsGenerator.generateStats('done', runnerId, statDateSixth));
+                should(doneStatsResponse.statusCode).be.eql(204);
+                
+
+                const getExportedReportResponse = await reportsRequestCreator.getExportedReport(testId, reportId, 'csv');
+                should(getExportedReportResponse.statusCode).be.eql(200);
+
+                const reportLines = getExportedReportResponse.text.split('\n');
+                
+                for (let index = 1; index < reportLines.length; index++){
+                    console.log(reportLines[index].split(","));
+                }
+
+                let splitData = [];
+                for (let i = 1; i < reportLines.length; i++){
+                    splitData.push(reportLines[i].split(","));
+                }
+
+                const DATA_ROW = "357.2,1042,1059,90.99,101";
```

Comment:
can you add here comments on what exactly you are testing, 
it will be great if the test can be parsed to JSON 
the important part here IMO is why it's 357.2,1042 etc?
I guess this was somehow calculated to be true
maybe it can be calculated in the test so if someone changes stats, the test will stay green?
will be much easier to understand

Reply:
I have used the data directly from the `statsGenerator`. Do let me know if that is alright. Thank you!